src/control/cmd/dmg/pool_test.go

```diff
@@ -161,6 +163,19 @@ func TestPoolCommands(t *testing.T) {
 	}
 	createACLFile(t, testACLFile, testACL)
 
+	// An existing file with nothing in it, for tests that need to verify no
+	// overwrite
+	testExistingFile := filepath.Join(tmpDir, "existing.txt")
+	f, err := os.Create(testExistingFile)
+	if err != nil {
+		t.Fatalf("Couldn't create test file: %s", testExistingFile)
+	}
+	f.Close()
+
+	// Subdirectory with no write perms
+	testNoPermDir := filepath.Join(tmpDir, "badpermsdir")
```

Comment:
This was the case i was thinking above. Does stat come back with an error here? if so you might wabt to differentiate between no perms and clobbering which I don’t think i saw above.

Reply:
Stat does return an error, but Create returns the same one in that case.